### ISSUE TITLE
Error occurs when connecting to rippled that has no closed ledger

### ISSUE THREAD

**Issue by tarukosu** [ripple/ripple-rest#393]
I started standalone rippled with the following command.
```
$ rippled -a
```
Next, I tried to connect ripple-rest to it but an error ocuured. 
```
2015-08-28T02:16:33.094Z - error: uncaughtException: invalid range 
```

This is because "complete_ledgers" is "2" and causes parse error.

When I ran this command, this ploblem didn't occur.
```
$ rippled ledger_accept
```

An appropriate error message would be helpful.

**Comment by clark800** [comment]
This was fixed in ripple-lib 0.13.0-rc5: https://github.com/ripple/ripple-lib/commit/9f9e76f8b933201651af59307135f67cfa7d60e8

ripple-rest just needs the dependency updated

**Comment by tarukosu** [comment]
Thank you for your information.
I'll update ripple-lib.
